sway-lib-std/src/chain/auth.sw

```diff
@@ -49,10 +50,10 @@ pub fn msg_sender() -> Result<Identity, AuthError> {
 fn get_coins_owner() -> Result<Identity, AuthError> {
```

Comment:
Orthogonal to the actual change, but should we drop the `get_` here as we did for the other function in this lib?

Reply:
yes!

Reply:
Please file an issue!

Reply:
done: 
https://github.com/FuelLabs/sway/issues/2219